docs/advanced/document-store/benchmarking.md

```diff
@@ -0,0 +1,122 @@
+# Benchmarking
+## Results
+
+The script `scripts/benchmarking.py` benchmarks DocArray's supported Document Stores in 6 different operations:
+* Create (indexing Documents)
```

Comment:
```suggestion
- **Create**: remove () and really explain what does that mean by create, what function do you call. Right now you are explain `create` by `indexing Documents`, but then what is indexing document? 
```

Same for all other ops, explain what exactly happens there.

Reply:
explanation added